tests/ci-install.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+NULL=
+: "${ci_docker:=}"
+: "${ci_in_docker:=}"
+: "${ci_suite:=jessie}"
+
+if [ $(id -u) = 0 ]; then
+    sudo=
+else
+    sudo=sudo
+fi
+
+if [ -n "$ci_docker" ]; then
+    sed \
+        -e "s/@ci_distro@/${ci_distro}/" \
+        -e "s/@ci_docker@/${ci_docker}/" \
+        -e "s/@ci_suite@/${ci_suite}/" \
+        < tests/ci-Dockerfile.in > Dockerfile
+    exec docker build -t ostree-ci .
+fi
+
+case "$ci_distro" in
+    (debian)
+        # httpredir.debian.org seems to be unreliable, use a CDN instead
+        sed -i -e 's/httpredir\.debian\.org/deb.debian.org/g' /etc/apt/sources.list
+        ;;
+esac
+
+case "$ci_suite" in
+    (jessie)
+        $sudo apt-get -y update
+        $sudo apt-get -y install apt-transport-https wget
+        wget -O - https://sdk.gnome.org/apt/debian/conf/alexl.gpg.key | $sudo apt-key add -
+        echo "deb [arch=amd64] https://sdk.gnome.org/apt/debian/ jessie main" | $sudo tee /etc/apt/sources.list.d/flatpak.list
+        ;;
+
+    (trusty|xenial)
+        $sudo apt-get -y update
+        $sudo apt-get -y install software-properties-common
+        $sudo add-apt-repository --yes ppa:alexlarsson/flatpak
```

Comment:
FWIW very shortly the libgsystem dep will be gone; I haven't submitted a PR yet for https://github.com/cgwalters/ostree/commits/glnx-the-final-round but I will after the other pending ones are done.